.env.test

```diff
@@ -1,6 +1,8 @@
 DEBUG=true
 SECRET_KEY='tests'
 DATABASE_URL='postgres://localhost/opencraft'
+INSTANCE_MYSQL_URL='mysql://root@localhost'
+INSTANCE_MONGO_URL='mongodb://localhost'
```

Comment:
@omarkhan Ok for me to keep them on the vagrant instance. I'm still a bit wary of the long term effects of that decision though, as even if it does work relatively well in the current case, it still blurs the boundaries between the instances environment and the instance manager's one. But we can wait and see how that goes.

One thing that I would ask though, is to add at least one integration test for this feature - when we add features, or bugfixes around it, if there is no existing integration test for this, it will make things harder for the person who has to add it then, because there would be no existing infrastructure for it. It will also ensure that we keep deciding, on a case by case basis, whether changes around this feature are better tested by a unit or an integration test.